MediaGalleryUi/Model/GetDetailsByAssetId.php

```diff
@@ -90,10 +90,23 @@ public function execute(int $assetId): array
             'size' => $asset->getSize(),
             'tags' => $this->getKeywords($asset),
             'source' => $asset->getSource() ? $this->sourceIconProvider->getSourceIconUrl($asset->getSource()) : null,
-            'content_type' => $asset->getContentType()
+            'content_type' => $this->getContentType($asset->getContentType()),
```

Comment:
```suggestion
            'content_type' => strtoupper(str_replace('image/', '', $asset->getContentType()),
```

Reply:
Agree with the suggestion but from my side, it will be good to make `strtoupper` in the `$this->getContentType()`